guides/common/modules/proc_synchronizing-smart-proxy-directly-from-red-hat-cdn.adoc

```diff
@@ -0,0 +1,16 @@
+[id="Synchronizing_Smart_Proxy_Directly_From_Red_Hat_CDN_{context}"]
+= Synchronizing {SmartProxy} Directly From Red Hat CDN
+
+You can use simplified alternate content sources to configure the {SmartProxy} to sync content directly from Red Hat CDN.
+
+.Procedure
+. In the {ProjectWebUI}, navigate to *Content* > *Alternate Content Sources*.
+. Click *Add Source* and set the *Source type* as *Simplified*.
+. Set the *Content type* to *Yum*.
+. In the *Name* field, enter a name for the alternate content source.
+. Optional: In the *Description* field, provide a description for the alternate content source.
+. Select {SmartProxies} that you want to sync directly from Red Hat CDN.
+. Optional: Select Use *HTTP proxies* if you want the ACS to use the {SmartProxyServer}’s HTTP proxy.
```

Comment:
select _or_ use

Reply:
So there is a list of smart proxies presented in the UI and the user has to _select_ which smart proxies should have the particular ACS.

Reply:
I meant like it's currently "Optional: Select Use" which seems wrong. Maybe "Use is part of the highlighting? Or drop either "select" or "use" in this sentence.

Reply:
Ah so the option is literally called "Use HTTP proxies":
![image](https://user-images.githubusercontent.com/14796566/232805754-2b97e190-46d5-4161-b926-480f594beebf.png)


Maybe change to "* Use HTTP proxies *" ?

Reply:
Ah that was done, all good then :)

Reply:
@ianballou @maximiliankolb haha yeah stupid typo on my part. Sorry! :)